UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs

```diff
@@ -160,7 +161,23 @@ static IEnumerator LoadAdmins()
 		//ensure any writing has finished
 		yield return WaitFor.EndOfFrame;
 		Instance.serverAdmins.Clear();
-		Instance.serverAdmins = new HashSet<string>(AccessFile.ReadAllLines(Instance.adminsPath));
+		var collectionOfAdmins = AccessFile.ReadAllLines(Instance.adminsPath);
+		if (collectionOfAdmins == null)
+		{
+			Loggy.Error("[PlayerList.Admin/LoadAdmins] - The Admins file is null, empty or broken!", Category.Admin);
```

Comment:
the log includes the caller now

Reply:
is it outlined in the same way in our standards guide?

Reply:
```
2024-10-30 00:32:14,829 ERROR: [ItemSpawn] [RandomItemSpot/SpawnRandomItems] - No items spawned, expected 1 on LavalandRandomMobSpawner items but received 0 SpawnRandomItems::RandomItemSpot.cs:67
````

You can change the format if you want, see Loggy code

Reply:
removed logs